bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -363,7 +363,7 @@
           end
 
           it "locks the updated gem to the latest version" do
-            expect(file.content).to include("business (1.13.0)")
+            expect(file.content).to include("business (1.18.0)")
```

Comment:
A bunch of the expected versions have changed after updating the rubygems index response stub and package info responses to match.